### ISSUE TITLE
Reproducibility of paper results

### ISSUE THREAD

**Issue by wave-transmitter** [ubc-vision/Prompting-Hard-Hardly-Prompting#9]
Congratulations on your work!

I am attempting to reproduce the results from your paper, but it seems some critical information necessary for reproducibility might be missing. Could you please provide additional details about the training process? Specifically:

1. For how many steps was each prompt optimized? Was it 500 steps, similar to this repository?

2. Similarly, for how many timesteps was the diffusion process applied to generate images from the optimized prompts? Was it 50 timesteps, similar to this repository?

3. How much time did the optimization take per image? Could you also share details about the GPU setup?

4. According to the paper: "We validate our results on 500 images from the validation sets across 5 seeds." Does this mean you randomly selected 500 images from each dataset for 5 folds? If so, is there any way to access the filenames of the selected images?

Thank you in advance for your help!

Thanks!

**Comment by s-mahajan** [comment]
Hi, Thanks for your interest in the work: 

-- you can use much less, empirically around (80 steps is sufficient). It of course depends on the image, whether the diffusion process has seen those concepts before or not

-- Yes, 50 steps to generate the image. For prompt inversion, as stated in the paper, it is 400-900 based on our analysis

-- It takes approximately 3 hours per image. 

-- Yes, the images are randomly selected from the val/test set of the COCO and SUN datasets.
